apps/fabric-website/src/components/IconGrid/IconGrid.tsx

```diff
@@ -56,36 +58,84 @@ export class IconGrid extends React.Component<IIconGridProps, IIconGridState> {
     );
   }
 
+  public async componentDidMount() {
+    // Load the icons async after mount to avoid them being bundled and parsed with the page,
+    // due to their large bundle size.
+    switch (this.props.iconType) {
+      case 'font':
+        // The explicit json-loader! import helps prevent this from being processed by TS
+        // (we want only webpack to handle it)
+        const fontIcons = (await import('json-loader!@uifabric/icons/lib/data/AllIconNames.json')) as IIconInfo[];
```

Comment:
This is using a dynamic `import()` statement to load the file. That's a standard ES6 feature, but we can't use it in component library code since we can't assume anything about consumers' loading/bundling setup. However that's not an issue for the website since we know it will be bundled using webpack, which supports dynamic imports.

The `json-loader!` prefix is special webpack syntax telling which webpack loader to use to process this file. Per [these docs](https://v4.webpack.js.org/loaders/json-loader/) webpack would already use `json-loader` for `.json` files by default, but in this case I wanted to prevent typescript from complaining about the import without resorting to `ts-ignore` comments. 

The other option there would be omitting the loader comment and enabling typescript's `resolveJsonModule` option for this project, but I'm pretty sure we tried that sometime in the past and it broke the website (in a way which only showed up in a production build) due to differences between how TS and webpack output JSON modules, or something like that.